cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/export/StateVariablesExport.java

```diff
@@ -436,6 +442,10 @@ private static void writeStatus(Network network, String cimNamespace, XMLStreamW
     }
 
     private static void writeConnectableStatus(Connectable<?> connectable, String cimNamespace, XMLStreamWriter writer, CgmesExportContext context) {
+        if (connectable instanceof TieLine && connectable.hasProperty(Conversion.CGMES_PREFIX_ALIAS_PROPERTIES + CgmesNames.TERMINAL + "_Boundary_1")
+                && connectable.hasProperty(Conversion.CGMES_PREFIX_ALIAS_PROPERTIES + CgmesNames.TERMINAL + "_Boundary_2")) {
```

Comment:
when writing flows we use the **alias** `TERMINAL_Boundary_1/2`, but here we are checking if the **property** with same name exists. Do we have both the alias and a property defined ?

Reply:
You're right! I'm adding the properties in `ACLineSegmentConversion` as they are the ones handled by merging at the moment.

Reply:
debugging a test case (CGMES cas 3 conformity BE+NL) I see that the following are defined:

```
properties
  CGMES.Terminal_1
  CGMES.Terminal_2
  CGMES.Terminal_Boundary_1
  CGMES.Terminal_Boundary_2
aliasesByType
  CGMES.Terminal2
  CGMES.Terminal1
```

I am checking all the values. I think the aliases for `Terminal1/2` are created automatically and maybe we should remove them ... let me check it.

Reply:
With the current version (commit [6e975a3](https://github.com/powsybl/powsybl-core/pull/2452/commits/6e975a306a807203d369502189d87a4ceaabb81c included), when we export the power flows for the tie lines we are using the aliases created automatically when preparing the export (`CGMES.Terminal1/2`, created in `CgmesExportContext::addIidmMappingsTerminal`), instead of the terminals read from CGMES input and stored in properties (`CGMES.Terminal_1`).

I think that we should either store the original terminal info in the alias (record in `CGMES.Terminal1` alias instead or in addition to saving it in `CGMES.Terminal_1` property, like we do for any generic connectable) or we avoid creating the auto aliases for tie lines in `CgmesExportContext::addIidmMappings...` (treating these lines as an special case).

I think the best option would be to set the aliases `Terminal1/2`, so we could have "uniform" treatment in other parts of the code. What do you think ?

Reply:
Yes, I agree in `addIidmMappings`, if there is a `CGMES.Terminal_1/2` property, we can set it in `CGMES.Terminal1/2`

Reply:
I will do the change and re-test it

Reply:
ah oops didn't see your comment in time sorry, i've already commit it....

Reply:
no problem, I was out.
but the way you fixed is not adding the alias by type to the tie line, because the alias already exists in the network index. I am tracing it ...

Reply:
the alias is already present in the index. It is stored in the map `idByAlias`, pointing to the id of the tie line. The attempt to add as new alias by type is silently ignored because it will point to the same object (the tie line). So we are are "silently" losing the alias by the new type.

Reply:
I think the aliases (at least for terminals 1) are added by id, without type, from the dangling line when the merged line is created. In the method `NetworkImpl::replaceDanglingLineByLine`, at the point `mergedLine.aliases.forEach(l::addAlias)`

Reply:
Proposal:
- rename method NetworkImpl::replaceDanglingLineByLine to NetworkImpl::createTieLinesFromMergedLines, 
- and take care of re-creating the proper aliases:
  * after mergedLine.aliases.forEach(l::addAlias), 
  * remove the (untyped) aliases of terminals and re-create them as alias with proper type

Reply:
Yes, there are issues in the merging of aliases but I don't think we should resolve them in this PR as it should be resolved in the improvements of the merging. It prevents doing the work twice!

Reply:
I made the changes proposed. It is a small fix in the implementation of tie line creation plus extending the unit tests ...

Reply:
At least we will have related information in the point where we are creating the tie lines